### DISCUSSION TITLE
How to create new window in v2

### DISCUSSION THREAD

**Discussion by dladeira** [tauri-apps/tauri#9601] [General]
Hey everyone,

I'm trying to migrate from v1 to v2. The documentation is very unfinished, but I found [this section](https://docs.rs/tauri/2.0.0-beta.16/tauri/window/struct.WindowBuilder.html) in the cargo documentation, which says that I should be able to create a new window like this:

```
let window = tauri::window::WindowBuilder::new(app, "main".to_string())
              .title("Localhost Example")
              .build()?;
```

However it throws an error: `struct 'WindowBuilder' is private`

Any help is greatly appreciated

**Comment by FabianLars** [comment]
The v1 WindowBuilder is `WebviewWindowBuilder` in v2, so this page: https://docs.rs/tauri/2.0.0-beta.16/tauri/webview/struct.WebviewWindowBuilder.html

This was done in preparation for multi-webview window support some time later.
